content.en/tools/nebula/_index.md

```diff
@@ -0,0 +1,108 @@
+![Nebula Logo](./nebula.png)
+
+# Nebula
+
+Nebula is a [libp2p](https://github.com/libp2p/go-libp2p-kad-dht) DHT crawler
+and monitor that tracks the liveliness of peers. The crawler connects
+to [DHT](https://en.wikipedia.org/wiki/Distributed_hash_table) bootstrap peers
+and then recursively follows all entries in
+their [k-buckets](https://en.wikipedia.org/wiki/Kademlia) until all peers have
+been visited. The crawler supports
+the [IPFS](https://ipfs.network), [Filecoin](https://filecoin.io), [Polkadot](https://polkadot.network/), [Kusama](https://kusama.network/), [Rococo](https://substrate.io/developers/rococo-network/), [Westend](https://wiki.polkadot.network/docs/maintain-networks#westend-test-network)
+networks [and more](https://github.com/dennis-tra/nebula/blob/a33a5fd493caaeb07e92ecc73c32ee87ae9e374f/pkg/config/config.go#L11).
+
+{{< button href="https://github.com/dennis-tra/nebula" >}}GitHub{{< /button >}}
+
+## Components
+
+Nebula is split into two components 1) a **crawler** and 2) a **monitor**. The
+crawler
+is responsible for discovering peers in the DHT and the monitor periodically
+connects to the peers to track their uptime.
+
+## How does it work?
+
+The Kademlia DHT network is a distributed system where each peer maintains a
+routing table containing other peers in the network with specific XOR distances
+to itself. Peers are grouped into "k-buckets" based on the number of shared
+prefix bits between their
+own [`PeerID`](https://docs.libp2p.io/concepts/peer-id/) and the remote PeerID.
+For instance, bucket
+0 includes nodes without any shared prefix bits, and each bucket contains a
+maximum of 20 entries.
+
+To begin the crawl process, Nebula connects to a set of bootstrap peers and
```

Comment:
How are the bootstrap peers chosen. Are they are standard set?

Reply:
added more information 👍